src/internal/hooks/useCapabilitiesSafe.ts

```diff
@@ -0,0 +1,23 @@
+import { useMemo } from 'react';
+import type { WalletCapabilities } from 'viem';
+import { useAccount } from 'wagmi';
+import { useCapabilities } from 'wagmi/experimental';
+import type { UseCapabilitiesSafeParams } from '../../types';
+
+export function useCapabilitiesSafe({
```

Comment:
I still feel like we should rename this `useCapabilities`, as the fact that we are not exporting this externally adding the word Safe just feels odd.

Reply:
I think it will make more sense once we add the functionality to debounce or stop subsequent calls to non-EIP5792 wallets on this hook

Reply:
(which isn't supported in the vanilla Wagmi hook - hence `unsafe`)

Reply:
I'd also say the vanilla Wagmi hook is `unsafe` because it does not return an empty object for non-EIP5792 wallets